scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -227,7 +228,7 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         {
             foreach (var file in Directory.GetFiles(Dirs.CorehostLocalPackages, "*.nupkg"))
             {
-                var hostBlob = $"{Channel}/Binaries/{CliNuGetVersion}/{Path.GetFileName(file)}";
+                var hostBlob = $"{Channel}/Binaries/{SharedFrameworkNugetVersion}/{Path.GetFileName(file)}";
```

Comment:
> What I'm worried about here is that the host versions and sharedframework versions are different.

Yup, they are different, but that doesn't matter so much.  The SharedFramework references the DotNetHost packages.  So SharedFX v1.0.0-rc3-002799 references DotNetHost v1.0.1-rc3-002799 (or in some cases it might reference the "locked" version).

So the idea here is that we will upload all the corresponding packages (SharedFX and DotNetHost) to the most consumable package's version folder, namely the SharedFX:  `1.0.0-rc3-002799`.  The DotNetHost packages that correspond to SharedFX `1.0.0-rc3-002799` will be in that folder, even though they have a different version number.

The CLI works the same way.  We add these DotNetHost packages into our SDK installer, and even though our installer has a different version, we still upload the installer to a folder with the installer's version number.  Since it is actually the thing people consume.

I don't think it would make sense to split the SharedFX and the DotNetHost packages into two different Azure folders corresponding to their versions.  We built them together - we put them in the same folder together.